packages/framework/presence/src/test/mockEphemeralRuntime.ts

```diff
@@ -5,13 +5,48 @@
 
 import { strict as assert } from "node:assert";
 
+import type { IAudience } from "@fluidframework/container-definitions";
 import type { ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
-import type { IQuorumClients, ISequencedClient } from "@fluidframework/driver-definitions";
-import { MockQuorumClients } from "@fluidframework/test-runtime-utils/internal";
+import type {
+	IClient,
+	IQuorumClients,
+	ISequencedClient,
+} from "@fluidframework/driver-definitions";
+import { MockAudience, MockQuorumClients } from "@fluidframework/test-runtime-utils/internal";
 
 import type { ClientConnectionId } from "../baseTypes.js";
 import type { IEphemeralRuntime } from "../internalTypes.js";
 
+/**
+ * Creates a mock {@link @fluidframework/container-definitions#IAudience} for testing.
+ */
+export function makeMockAudience(clientIds: string[]): IAudience {
```

Comment:
How do we coordinate with quorum? audience should always be a superset of quorum. (subset should have identical information.)

Reply:
They are populated using the same clients so they currently will have identical information.

Reply:
But they don't. Audience has odd clients with "read" mode, but quorum are always "write". And the user information is different.
Let's not fix it now. Can do after.